coordinator/internal/config/config.go

```diff
@@ -50,13 +50,19 @@ type Config struct {
 	Auth          *Auth            `json:"auth"`
 }
 
+// CircuitConfig circuit items.
+type CircuitConfig struct {
+	ParamsPath string `json:"params_path"`
+	AssetsPath string `json:"assets_path"`
+	Version    string `json:"version"`
+}
+
 // VerifierConfig load zk verifier config.
 type VerifierConfig struct {
-	ForkName     string `json:"fork_name"`
-	MockMode     bool   `json:"mock_mode"`
-	ParamsPath   string `json:"params_path"`
-	AssetsPathLo string `json:"assets_path_lo"` // lower version Verifier
-	AssetsPathHi string `json:"assets_path_hi"` // higher version Verifier
+	ForkName           string         `json:"fork_name"`
```

Comment:
Could we also remove this if we use versions to determine which libzkp is called?